.github/workflows/docs.yml

```
@@ -0,0 +1,34 @@
+name: Docs
+
+on:
+  push:
+    branches: main
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout 🏷️
+        uses: actions/checkout@v2
+
+      - name: Set up Python 🐍
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.6'
```

```suggestion
          python-version: '3.9'
```

What about using a more recent python so we don't have to change it in the near future?

---

It makes more sense for me to keep the CI to the lowest version supported by the package (we have `python_requires = >=3.6`)